src/chrome_extension/manifest.json

```diff
@@ -17,18 +17,13 @@
   "permissions": [
     "tabs","contextMenus"
   ],
-  "background": {
-    "scripts": ["js/background.js"]
-  },
   "content_scripts": [
     {
       "matches": ["https://www.4628.jp/*"],
       "exclude_globs": ["*markdays=*"],
       "js": [
-        "js/jquery.min.js",
-        "js/content_script.js"
+        "content.min.js"
       ],
-      "css": ["css/app.css"],
       "run_at": "document_end"
```

Comment:
I confirmed that I made 「src/chrome_extension/css/app.css」 to 「src/common/js/timesheet/index.scss.

And I also confirmed that it was taken over by 「/chrome_extension/content.min.js」 after 「yarn deploy」.

If so, do not need the 「https://github.com/umesan/overtime-application/blob/webpack/src/chrome_extension/css/app.css」 file, please delete